tests/unittest_brain_numpy_ma.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2021 hippo91 <guillaume.peillex@gmail.com>
+
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+import unittest
+
+try:
+    import numpy  # pylint: disable=unused-import
+
+    HAS_NUMPY = True
+except ImportError:
+    HAS_NUMPY = False
+
+from astroid import builder
+
+
+@unittest.skipUnless(HAS_NUMPY, "This test requires the numpy library.")
+class BrainNumpyMaTest(unittest.TestCase):
```

Comment:
Why not use `pytest.skip`? The class name should be changed to `TestBrainNumpyMa`. `pytest` requires `Test` as prefix.